client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -1060,9 +1063,9 @@ const ConnectedPlanFeatures2023Grid = connect(
 				);
 			}
 
-			const rawPriceAnnual =
+			const maybeDiscountedPriceFullTerm =
```

Comment:
For lack of a better term. Just the existence of a bad/strange name is probably a signal of underlying complexities that need resolution. Technically, we have a "maybe" discounted price for the plan's full term (annual or biannual) that may or may not differ from `rawPrice`. 

We can tweak the naming a bit (the "term" can be more specific to annual/biannual), but that aside, it might be better to leave the dragon lurking and refactor more holistically at some point.